File: ShareAGame.DataAccess/Models/CreateNewGameDto.cs
using ShareAGame.DataAccess.Entities;
using System;

namespace ShareAGame.DataAccess.Models
{
  public class CreateNewGameDto
  {
        public string Name { get; set; }
        public bool IsDigital { get; set; }
        public DateTimeOffset ReleaseDate { get; set; }
        public string ImageUrl { get; set; }
        public virtual Publisher Publisher { get; set; }
        public GamePlatform GamePlatform { get; set; }
    }
}

File: SharAGameAPI/ClientApp/src/app/models/request/create-new-game-dto.ts
import { GamePlatfromEnum } from '../enums/game-platform-enum';

export interface CreateNewGameDto {
  name: string;
  isDigital: boolean;
  releaseDate: Date;
  imageUrl: string;
  publisherId: string;
  gamePlatform: GamePlatfromEnum;
}
